bpf/lib/lb.h

```diff
@@ -547,6 +641,13 @@ static __always_inline int lb6_local(const void *map, struct __ctx_buff *ctx,
 	 * session we are likely to get a TCP RST.
 	 */
 	if (!(backend = lb6_lookup_backend(ctx, state->backend_id))) {
+// NOTE(brb): Can't enable the removal, as otherwise the verifier hits >1mln
+// insn limit. Hovewer, the removal of the affinity in this case is just an
+// optimization.
```

Comment:
nit: kernel style.
Should we then enable this when ! HAVE_LARGE_INSN_LIMIT ?

Reply:
I will try it. BTW, the troublesome (bisected) change in the verifer was introduced after `HAVE_LARGE_INSN_LIMIT`, right?

Reply:
Checked that it works on 4.19.